### ISSUE TITLE
Artifactory Helm install failed - remove-lost-found init error

### ISSUE THREAD

**Issue by eladh** [jfrog/charts#247]
**Is this a request for help?**:

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG

**Version of Helm and Kubernetes**:
Kubernetes - 1.11.7-gke.4

Helm Client: &version.Version{SemVer:"v2.12.3", GitCommit:"eecf22f77df5f65c823aacd2dbd30ae6c65f186e", GitTreeState:"clean"}
Helm Server: &version.Version{SemVer:"v2.12.3", GitCommit:"eecf22f77df5f65c823aacd2dbd30ae6c65f186e", GitTreeState:"clean"}

**Which chart**:
artifactory-7.12.7

**What happened**:
```
remove-lost-found:
    Container ID:  docker://6a267f71dc8c3016aa9bb164568c243b6a8a6b6db5f4db46c302c580e70db538
    Image:         alpine:3.8
    Image ID:      docker-pullable://alpine@sha256:899a03e9816e5283edba63d71ea528cd83576b28a7586cf617ce78af5526f209
    Port:          <none>
    Host Port:     <none>
    Command:
      sh
      -c
      rm -rfv /var/opt/jfrog/artifactory/lost+found /var/opt/jfrog/artifactory/data/.lock
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Fri, 15 Mar 2019 10:34:29 +0200
      Finished:     Fri, 15 Mar 2019 10:34:29 +0200
    Ready:          False
    Restart Count:  7

```
**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):
helm install --name artifactory jfrog/artifactory

**Anything else we need to know**:
Running the same command under artifactory-7.12.5 working without any problem

**Comment by eladh** [comment]
@eldada - i also see a similar behaviour in distribution 


```
2019-03-15 10:56:50   [16 wrapper.sh] Testing directory (/opt/jfrog/distribution-distribution) has read/write permissions
2019-03-15 10:56:50   [24 wrapper.sh] /opt/jfrog/distribution-distribution has read/write permissions for distribution
2019-03-15 10:56:50   [16 wrapper.sh] Testing directory (/var/opt/jfrog/distribution) has read/write permissions
./wrapper.sh: line 21: /var/opt/jfrog/distribution/test-permissions: Permission denied
2019-03-15 10:56:50   [37 wrapper.sh] ###########################################################
2019-03-15 10:56:50   [38 wrapper.sh] /var/opt/jfrog/distribution is NOT writable!
2019-03-15 10:56:50   [39 wrapper.sh] Directory: /var/opt/jfrog/distribution, permissions: 755, owner: root, group: root
2019-03-15 10:56:50   [40 wrapper.sh] Mounted directory must be writable by user 'distribution' (id 1020)
2019-03-15 10:56:50   [41 wrapper.sh] ###########################################################
2019-03-15 10:56:50   [65 wrapper.sh] ERROR: Directory /var/opt/jfrog/distribution has bad permissions for user 'distribution'
```

**Comment by danielezer** [comment]
This still isn't working. There's issue with the permissions on the filesystem. The securityContext in the container level does not apply to the volumes. I am trying to fix that, might need to revert though

**Comment by danielezer** [comment]
fixed in #249
